lib/quiz/quiz_scroller_pagger.dart

```diff
@@ -0,0 +1,179 @@
+import 'package:flutter/material.dart';
+import 'dart:math';
+import 'dart:ui' as ui;
+import 'dart:async';
+import 'card_list.dart';
+
+class Quizscroller_pagger extends StatefulWidget {
```

Comment:
Pay attention to spelling: Not pagger, pager.
Also in Dart you never use _ in class names. So - this should be QuizScrollerPager